eos-phone-home

```diff
@@ -460,7 +459,8 @@ if __name__ == '__main__':
 
     args = p.parse_args()
     is_debug = args.debug or args.force
+    logging.basicConfig(level='DEBUG' if args.debug else 'INFO')
```

Comment:
I've always seen this used with the constants defined in `logging` itself, e.g. `logging.DEBUG`
Also, maybe you should use `is_debug` instead of `args.debug` here